department-of-veterans-affairs/abd-vro
# Don't always use jdbc for postgres URLs

## What was the problem?

Historically, all of our database connections occurred through Java applications so the `jdbc:` prefix was included in the URLs. But we want to be more general than this.

Associated tickets or Slack threads:
- #1845 

## How does this fix it?[^1]


## How to test this PR
-  Integration tests will confirm access to DB using the new setup

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
